search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -75,12 +76,83 @@ function displayTopResultsForCurrentTrend(countryCode) {
         countryData[0].averageSentiment + '<br>';
 
     for (let i = 0; i < results.length; i++) {
-      resultElement.innerHTML += '<a href=' + results[i].link + '>' +
-        results[i].htmlTitle + '</a><br>' + results[i].snippet + '<br>'
-        + 'Sentiment Score: ' + results[i].score + '<br>';
+      let searchResultIndex = i + 1;
+      resultElement.innerHTML += searchResultIndex + '. ' + '<a href=' + 
+          results[i].link + '>' + results[i].htmlTitle + '</a><br>' + 
+          results[i].snippet + '<br>' + 'Sentiment Score: ' + results[i].score + 
+          '<br>';
     }
     resultElement.innerHTML += '<i>Last updated on ' + date.toString() +
       '<i><br>';
   }
 }
 
+
+/** 
+ * Displays the sentiment chart in a country for current search trend on modal. 
+ * @param {string} countryCode Two letter country code for selected country.
+ */
+function displaySentimentChartForCurrentTrend(countryCode) {
+  let topic = getCurrentTrend;
+  let topicData = getCurrentCustomSearchData();
+  let date = new Date(topicData.timestamp);
+  let chartElement = document.getElementById('sentiment-chart-tab');
+  chartElement.innerHTML = '';
+
+  let countryData = topicData.countries
+      .filter(countries => countries.country === countryCode);
+
+  // Handle case where there are no results.
+  if (countryData.length === 0) {
```

Comment:
```suggestion
  if (countryData.length === 0 ||
      countryData[0].averageSentiment === NO_RESULTS_DEFAULT_SCORE)) {
```

And add this at the top. 
```
// The default score assigned to countries with no search results.
const NO_RESULTS_DEFAULT_SCORE = -500;
```

This is all copied form the user-search branch. Handle's case where there are no search results for that country and therefore the sentiment score is -500.

Reply:
Bumping this again.